tensorflow_quantum/core/serialize/serializer_test.py

```diff
@@ -407,6 +408,29 @@ def _get_valid_pauli_proto_pairs():
     return pairs
 
 
+def _get_valid_projector_proto_pairs():
+    """Generate valid projectorsum proto pairs."""
+    q0 = cirq.GridQubit(0, 0)
+    q1 = cirq.GridQubit(1, 0)
```

Comment:
Maybe throw in some `LineQubit`s here too ?

Reply:
Done.